composer.json

```diff
@@ -49,7 +46,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "scripts": {
-        "analyze": "phpstan analyze --no-progress",
+        "analyze": "psalm",
```

Comment:
minor: I think we should standarise the way how we are adding static analysis to the project so it is consistent and we can have one simple contribution guide, see how it has been done in laminas-cli:

https://github.com/laminas/laminas-cli/blob/0.2.x/composer.json#L64-L71

Probably the same we should do in Travis CI configuration.